include/psp2/screenshot.h

```diff
@@ -1,5 +1,5 @@
 /**
- * \usergroup{SceScreenShot}
+ * \usergroup{SceScreenshot}
```

Comment:
@frangarcj capitalized the second "s" of the "Screenshot" occurrences, so the change I'm requesting here is in line with it.